main.go

```diff
@@ -158,14 +160,14 @@ func main() {
 	}
 }
 
-func runPublish(expo Expo, cfg Config) {
+func runPublish(expo Expo, cfg Config) error {
 	//
 	// Logging in the user to the Expo account
 	fmt.Println()
 	log.Infof("Login to Expo")
 	{
 		if err := expo.login(cfg.UserName, cfg.Password); err != nil {
-			failf("Failed to log in to your provided Expo account, error: %s", err)
+			return fmt.Errorf("failed to log in to your provided Expo account, error: %s", err)
```

Comment:
One small thing: it is not best practice anymore to use ", error:" you can just leave the error out.